module/shardingManager.ts

```diff
@@ -563,7 +563,10 @@ export async function handleDiscordPayload(
       }
 
       if (data.t === "PRESENCE_UPDATE") {
-        return eventHandlers.presenceUpdate?.(data.d as PresenceUpdatePayload);
+        const payload = data.d as PresenceUpdatePayload;
+        const oldPresence = cache.presences.get(payload.user.id) ?? null;
+        cache.presences.set(payload.user.id, payload);
+        return eventHandlers.presenceUpdate?.(oldPresence, payload);
```

Comment:
```suggestion
        return eventHandlers.presenceUpdate?.(payload, oldPresence);
```
Can we reverse this and make payload first. Consistency wise with the rest of the lib like memberUpdate, roleUpdate etc all the update events have the latest data first and the cached data as last.

Example:
`guildMemberUpdate?.(guild, member, cachedMember);`